djstripe/enums.py

```diff
@@ -754,6 +754,8 @@ class SubscriptionScheduleStatus(Enum):
 
 
 class SubscriptionStatus(Enum):
+    all = _("All")
```

Comment:
? what is this?

Reply:
@jleclanche `Subscription` status has additional enums that become relevant when calling `list()`. it's mentioned [here](https://stripe.com/docs/api/subscriptions/list#list_subscriptions-status)

Reply:
@arnav13081994 Those aren't subscription statuses (as in, possible statuses the subscription can have), they are special values which are specific to the list api and do not belong in the enum.

The enum describes the possible values *in the database*. This list enum is technically separate, but it's not worth bothering with it IMO. We already maintain way too much of the Stripe API outside the SDK and it's becoming unmanageable.

Reply:
I understand that. The only other option would be to use a custom handler for list_Leary's as it's done for other models but Subscription list is used in syncing other items too so other handlers will need to be modified too. 

What do you suggest @jleclanche?